spec/lib/sufia/import/collection_builder_spec.rb

```diff
@@ -15,7 +15,7 @@
   end
 
   it "creates a collection with metadata and permissions" do
```

Comment:
I wonder if you created multiple collections within the same test if you would see the error?